src/lib/containers/SearchV2.tsx

```diff
@@ -135,93 +136,101 @@ class Search extends React.Component<InternalSearchProps, SearchState> {
               <FontAwesomeIcon
-                className="SRC-primary-text-color"
-                icon="times"
+                className="SearchV2__searchbar__searchicon"
+                size={'sm'}
+                icon={'search'}
               />
-            </button>
-          )}
-        </form>
-        <div
-          className={
-            show
-              ? 'SearchV2__form-container'
-              : 'SearchV2__form-container hidden'
-          }
-        >
-          <form ref={this.radioFormRef} className="SearchV2__column-select">
-            <p className="deemphasized">
-              <i> Search In Field: </i>
-            </p>
-            {data?.columnModels?.map((el, index) => {
-              const name = el.name
-              const displayName = unCamelCase(el.name, facetAliases)
-              const selectedColumn = this.state.columnName
-              const isSelected =
-                (selectedColumn === '' && index === 0) ||
-                selectedColumn === name
-              return (
-                <div className="radio">
-                  <label>
-                    <span>
-                      <input
-                        id={name}
-                        type="radio"
-                        value={name}
-                        checked={isSelected}
-                        onClick={() => {
-                          this.setState({
-                            columnName: name,
-                          })
-                        }}
-                      />
-                      <span>{displayName}</span>
-                    </span>
-                  </label>
-                </div>
-              )
-            })}
-          </form>
+              <input
+                onChange={this.handleChange}
+                onClick={() => {
+                  this.setState({
+                    show: true,
+                  })
+                }}
+                placeholder="Enter Search Terms"
+                value={searchText}
+                type="text"
+              />
+              {this.state.searchText.length > 0 && (
+                <button
+                  className="SearchV2__searchbar__clearbutton"
+                  type="button"
+                  onClick={() => {
+                    this.setState({
+                      searchText: '',
+                    })
+                  }}
+                >
+                  <FontAwesomeIcon
+                    className="SRC-primary-text-color"
+                    icon="times"
+                  />
+                </button>
+              )}
+            </form>
+          </div>
+        </CSSTransition>
+        <div className="SearchV2__dropdown_pos">
+          <CSSTransition
+          in={show}
+          classNames="SearchV2__animate_dropdown"
+          timeout={{ enter: 0, exit: 300 }}
+          >
+
+              <form ref={this.radioFormRef} className="SearchV2__column-select SearchV2__animate_height">
```

Comment:
@emmanmills if you could add `.vscode/settings.json` to your workspace with the following contents `settings.json` then you should get auto formatting.
```
{
  "editor.formatOnSave": true,
  "eslint.alwaysShowStatus": true
}
```

Reply:
Cool! Done.